notebook/static/notebook/js/notificationarea.js

```diff
@@ -250,7 +250,7 @@ define([
             link.id = 'favicon';
             link.type = 'image/x-icon';
             link.rel = 'shortcut icon';
-            link.href = src;
+            link.href = utils.url_path_join(utils.get_body_data('baseUrl'), src);
```

Comment:
The paths passed into this function have a leading `/` - is this OK, or do they need to be changed as you've changed the other path below?